File: CS/A2404/program.cs
using System;
using DevExpress.Xpo;
using DevExpress.Xpo.DB;
using System.Windows.Forms;
using A2404;

namespace DXSample {
    public class Program {
        [STAThread]
        static void Main() {
            XpoDefault.ConnectionString = AccessConnectionProvider.GetConnectionString(@"..\..\data.mdb");
            Application.EnableVisualStyles();
            Application.Run(new Form1());
        }
    }
}

File: VB/A2404/program.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Xpo.DB
Imports System.Windows.Forms
Imports A2404

Namespace DXSample
	Public Class Program
		<STAThread> _
		Shared Sub Main()
			XpoDefault.ConnectionString = AccessConnectionProvider.GetConnectionString("..\..\data.mdb")
			Application.EnableVisualStyles()
			Application.Run(New Form1())
		End Sub
	End Class
End Namespace